openupgrade_scripts/scripts/crm_iap_mine/15.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,116 @@
+from openupgradelib import openupgrade
+
+_xmlid_renames = [
+    (
+        "crm_iap_mine.crm_iap_lead_industry_30",
```

Comment:
Are we sure?

I would have though xml_ids in a rename would be `crm_iap_lead.x -> crm_iap_mine.x` . More for my knowledge, as maybe this happens earlier in process.

Reply:
@gdgellatly whoops my mistake, indeed it would be crm_iap_lead.x -> crm_iap_mine.x, i 'll fix right away